### ISSUE TITLE
Deploy error:  NoMethodError: undefined method `<' for nil:NilClass

### ISSUE THREAD

**Issue by medir** [Mixd/wp-deploy#130]
Hi,

When I try to deploy remotely or locally I'm getting stuck with the next error:

```
bundle exec cap staging wp:setup:local --trace
** Invoke staging (first_time)
** Execute staging
** Invoke load:defaults (first_time)
** Execute load:defaults
** Invoke wp:setup:local (first_time)
** Execute wp:setup:local
cap aborted!
NoMethodError: undefined method `<' for nil:NilClass
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/sshkit-1.7.1/lib/sshkit/formatters/pretty.rb:8:in `write'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/sshkit-1.7.1/lib/sshkit/backends/local.rb:59:in `block in _execute'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/sshkit-1.7.1/lib/sshkit/backends/local.rb:58:in `tap'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/sshkit-1.7.1/lib/sshkit/backends/local.rb:58:in `_execute'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/sshkit-1.7.1/lib/sshkit/backends/local.rb:32:in `capture'
/home/xyz/spl-repos/wp-festela/lib/capistrano/tasks/wp.cap:102:in `block (4 levels) in <top (required)>'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/sshkit-1.7.1/lib/sshkit/backends/local.rb:15:in `instance_exec'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/sshkit-1.7.1/lib/sshkit/backends/local.rb:15:in `run'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/capistrano-3.4.0/lib/capistrano/dsl.rb:59:in `run_locally'
/home/xyz/spl-repos/wp-festela/lib/capistrano/tasks/wp.cap:84:in `block (3 levels) in <top (required)>'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/task.rb:240:in `block in execute'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/task.rb:235:in `each'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/task.rb:235:in `execute'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/task.rb:179:in `block in invoke_with_call_chain'
/home/xyz/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/monitor.rb:214:in `mon_synchronize'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/task.rb:172:in `invoke_with_call_chain'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/task.rb:165:in `invoke'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:150:in `invoke_task'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:106:in `block (2 levels) in top_level'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:106:in `each'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:106:in `block in top_level'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:115:in `run_with_threads'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:100:in `top_level'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:78:in `block in run'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:176:in `standard_exception_handling'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/rake-10.4.2/lib/rake/application.rb:75:in `run'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/capistrano-3.4.0/lib/capistrano/application.rb:15:in `run'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/gems/capistrano-3.4.0/bin/cap:3:in `<top (required)>'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/bin/cap:22:in `load'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/bin/cap:22:in `<main>'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/bin/ruby_executable_hooks:15:in `eval'
/home/xyz/.rvm/gems/ruby-2.4.0@wptest1/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => wp:setup:local
```
My development environment:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.1 LTS
Release:	16.04
Codename:	xenial

$ ruby -v
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-linux]

$ wp --info
PHP binary:	/usr/bin/php5.6
PHP version:	5.6.30-1+deb.sury.org~xenial+1
php.ini used:	/etc/php/5.6/cli/php.ini
WP-CLI root dir:	phar://wp-cli.phar
WP-CLI packages dir:	
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	1.1.0

$ php -v
PHP 5.6.30-1+deb.sury.org~xenial+1 (cli) 
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
```

Finally I found next solution:
1 Delete Gemfile.lock
2 Fix capistrano gem to 3.4.0:
3 Downgrade colorize gem to 0.7.4
4 bundle install

```
# Gemfile
gem 'capistrano', '3.4.0'
gem 'colorize', '0.7.4'
gem 'capistrano-slackify', '~> 2.2.0'
```
Hope it can be useful to somebody :)

**Comment by fredbaa** [comment]
Having the same issue with this. Using ruby 2.4.0

**Comment by lustremedia** [comment]
Yeah the project should really be upgraded to the latest Capistrano 3.8, looks a bit like it has been abandoned ...

**Comment by lukewhitehouse** [comment]
@medir @fredbaa you can do a very quick fix for this by putting Cap up to 3.4.1. Or, wait till the next release (see dev branch!)

**Comment by yeahsmaggy** [comment]
As @lustremedia said I just changed capistrano version in Gemfile to 3.8 and it worked.
